app-arch/unarr/unarr-1.0.0.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="Decompression library for RAR, TAR, ZIP and 7z archives"
+DESCRIPTION="
+unarr originated as a port of the RAR extraction features from \
+The Unarchiver project required for extracting images from comic \
+book archives (.cbr). It was written as an alternative to \
+libarchive which didn't have support for parsing filters or solid \
+compression at the time.
"
+HOMEPAGE="https://github.com/sumatrapdfreader/sumatrapdf/tree/master/ext/unarr"
+COPYRIGHT="The Unarchiver project (https://code.google.com/p/theunarchiver/) \
+Simon Bünzli (zeniko at gmail.com, http://www.zeniko.ch/#SumatraPDF)"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/zeniko/unarr/archive/d1be8c43a82a4320306c8e835a86fdb7b2574ca7.zip"
+CHECKSUM_SHA256="91a4c75a62d14513f95a193668565124acae1c3b6b39384212c6313821b92c4f"
+SOURCE_DIR="unarr-d1be8c43a82a4320306c8e835a86fdb7b2574ca7"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	unarr$secondaryArchSuffix = $portVersion
+	cmd:unarr$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libunarr$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -e s/-lm//g -i Makefile
+}
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp build/debug/unarr-test $binDir/unarr
+	mkdir -p $developDir
```

Comment:
worked on the recipe a bit and uploaded a new one to my fork, maybe you could take a look at it?
https://github.com/Begasus/haikuports/commit/a0682402fb56055995f5b0a5dfc8fbee16b52a32#diff-d22ae810e98af09cafc5bd55e0af692d